model/__init__.py

```diff
@@ -1 +1,3 @@
+from .base_code_model import BaseCodeModel
```

Comment:
`BaseCodeModel` used as an abstract class, so it shouldn't be visible for users. We can remove it from here or use `__all__`